.github/workflows/comment_bot.yml

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Comment-bot
+
+on:
+  issue_comment:
+    types:
+      - created
+      - edited
+
+jobs:
+  autotune:
+    name: "Fixup pre-commit formatting"
+    if: startsWith(github.event.comment.body, '@github-actions autotune')
```

Comment:
I would maybe keep a "pre-commit" specific command 

(we can always later have other bot actions, and name something combined "autotune", but for now it's just pre-commit)